packages/console/src/pages/Applications/components/CreateForm/index.tsx

```diff
@@ -111,7 +110,8 @@ function CreateForm({
                 onChange={onChange}
               >
                 {Object.values(ApplicationType)
-                  .filter((value) => isDevFeaturesEnabled || value !== ApplicationType.Protected)
+                  // Protected app should not show up in the creation modal
+                  .filter((value) => value !== ApplicationType.Protected)
```

Comment:
nit: use a whitelist?

Reply:
Good call. Refactored to only show the 4 types: native, spa, traditional and m2m